### ISSUE TITLE
Buttons collide in RTL with fill setting

### ISSUE THREAD

**Issue by bmattb** [swissspidy/web-stories-wp#9809]
## Bug Description

Originally reported by @spacedmonkey 

> RTL + fill setting, first page is clipped. 

<img width="267" alt="image21" src="https://user-images.githubusercontent.com/58738530/143289236-40fbf92e-9b97-4f47-b1d8-7b2fe92f36d9.png">

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

**Comment by miina** [comment]
@spacedmonkey Could you specify the steps to reproduce? I can see the issue from the screenshot (that the buttons are overlapping), at the same time, the text says something else:
> RTL + fill setting, first page is clipped.

One the screenshot, it doesn't look like RTL, it's not Fill (it's Fit), and there's no first page being clipped. So the screenshot and the text don't seem to have much to do with each other. Was the text incorrectly copied from the sheet?

**Comment by spacedmonkey** [comment]
<img width="1792" alt="Screenshot 2021-11-25 at 13 23 14" src="https://user-images.githubusercontent.com/237508/143449275-b760009d-1051-449e-803e-d7e771257ff1.png">
<img width="1792" alt="Screenshot 2021-11-25 at 13 22 49" src="https://user-images.githubusercontent.com/237508/143449284-e5c0cdd2-72f4-44df-b02b-122c9cd7aba5.png">

This are exact screenshots. It is worth noting that the height of my browser window is pretty low. Either way I think it should fail a little more graceful than this. 

Worth noting, this is not RTL.

**Comment by miina** [comment]
Thanks for specifying.

I'm still confused about where this came from and how is that related:
> RTL + fill setting, first page is clipped.

Closing this issue as a duplicate of #9808 as the title of this issue is irrelevant to the real issue which should get resolved in #9808 

I'll add your screenshots to that issue instead, @spacedmonkey
